01-numpy.html

```diff
@@ -68,17 +68,23 @@ <h2 id="learning-objectives" class="objectives objectives">Learning Objectives</
 <pre class="output"><code>weight in kilograms is now: 57.5</code></pre>
 <p>As the example above shows, we can print several things at once by separating them with commas.</p>
 <p>If we imagine the variable as a sticky note with a name written on it, assignment is like putting the sticky note on a particular value:</p>
-<p><img src="img/python-sticky-note-variables-01.svg" alt="Variables as Sticky Notes" /></p>
+<div class="figure">
+<img src="fig/python-sticky-note-variables-01.svg" alt="Variables as Sticky Notes" /><p class="caption">Variables as Sticky Notes</p>
```

Comment:
Sorry that I didn't understand it correctly.

This is due one of the Pandoc's default extensions to Markdown. If you want we can disable this extension

~~~
$ pandoc -t html -f markdown-implicit_figures<<EOF 
![foo](bar.jpg) 
EOF
<p><img src="bar.jpg" alt="foo" /></p>
~~~

or do it for each case when need:

~~~
$ pandoc -t html <<EOF
![foo](bar.jpg)\  
EOF
<p><img src="bar.jpg" alt="foo" /> </p>
~~~

There is one white space right after the `\`.